app/styles/utils.scss

```diff
@@ -0,0 +1,71 @@
+$units: 0, 1, 2, 3, 4;
```

Comment:
I would prefer to add tailwind, rather than maintaining our own custom utils. What would your thoughts be on that?